lib/generators/action_view_preview/action_view_preview_generator.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "rails/generators"
+
+class ActionViewPreviewGenerator < Rails::Generators::NamedBase
+  source_root File.expand_path('templates', __dir__)
+
+  argument :name, type: :string, required: true, banner: 'NAME'
+  argument :actions, type: :array, default: [], banner: 'action action'
+
+  def execute
+    actions.map!(&:underscore)
+
+    template 'test/views/previews/%file_name%_preview.rb'
```

Comment:
Would it be possible to use the path from `ActionViewPreview.preview_path`? There's a [config](https://github.com/lcampanari/action_view_preview#configuration) to change the preview path (`test/views/previews` is the default value).

Reply:
Yeah! Good Idea! added.